### PR TITLE
🔀🔖 4.1.10-p1

### PR DESCRIPTION
**Pull request by Suchty112**
🚑[MissionHelper] Hovertip bug fix

### CODE REVIEW

**Review by Ron31 — APPROVED** [review]

File: src/modules/missionHelper/i18n/de_DE.root.json [hunk 1]
```diff
@@ -53,6 +53,10 @@
 			"description": "Blendet ELW 1 komplett aus und zeigt nur ELW 2 an.",
 			"title": "ELW 1 ausblenden"
 		},
+		"hoverTip": {
+				"description": "Tooltips für die Buttons wie der Overlay-Button",
```
Comment (KokosnussDEV) [inline src/modules/missionHelper/i18n/de_DE.root.json:57]: Are these indentations right?

File: src/modules/missionHelper/i18n/en_GB.root.json [hunk 2]
```diff
@@ -44,6 +44,10 @@
 			"description": "Hides Battalion Chief completely and shows only MCV.",
 			"title": "Hide Battalion Chiefs"
 		},
+		"hoverTip": {
+				"description": "Tooltips for as example the overlay button.",
```
Comment (KokosnussDEV) [inline src/modules/missionHelper/i18n/en_GB.root.json:48]: Again, are the indentations right?

Comment (Suchty112) [reply src/modules/missionHelper/i18n/en_GB.root.json:48]: they are then set correctly during the build :)

File: src/modules/missionHelper/i18n/de_DE.root.json (hunk 1 above)
Comment (Suchty112) [reply src/modules/missionHelper/i18n/de_DE.root.json:57]: they are then set correctly during the build :)

**Review by KokosnussDEV — APPROVED** [review]

**Pull request merged**